intro-HPC/ch_web_portal.tex

```diff
@@ -0,0 +1,217 @@
+        \item use the \emph{Delete} button to (\strong{permanently!}) remove the selected files and directories;
+    \end{itemize}
+\end{itemize}
+
+For more information, see aslo \url{https://www.osc.edu/resources/online_portals/ondemand/file_transfer_and_management}.
+
+
+\subsection{Job management}
+
+Via the \emph{Jobs} menu item, you can consult your active jobs or submit new jobs using the \emph{Job Composer}.
+
+For more information, see the sections below as well as
+\url{https://www.osc.edu/resources/online_portals/ondemand/job_management}.
+
+\subsubsection{Active jobs}
+
+To get an overview of all your currently active jobs,
+use the \emph{Active Jobs} menu item under \emph{Jobs}.
+
+A new browser tab will be opened that shows all your current queued and/or running jobs:
+
+\begin{center}
+    \includegraphics*[scale=0.5]{ood_active_jobs}
+\end{center}
+
+You can control which jobs are shown using the \emph{Filter} input area,
+or select a particular cluster from the drop-down menu \emph{All Clusters}, both at the top right.
+
+Jobs that are still queued or running can be deleted using the red button on the right.
+
+Completed jobs will also be visible in this interface, but only for a short amount of time after they have stopped running.
+
+For each listed job, you can click on the arrow ($>$) symbol to get a detailed overview of that job,
+and get quick access to the corresponding output directory (via the \emph{Open in File Manager} and \emph{Open in Terminal} buttons at the bottom of the detailed overview).
+
+\subsubsection{Job composer}
+
+To submit new jobs, you can use the \emph{Job Composer} menu item under \emph{Jobs}.
+This will open a new browser tab providing an interface to create new jobs:
+
+\begin{center}
+    \includegraphics*[scale=0.35]{ood_job_composer}
+\end{center}
+
+This extensive interface allows you to create jobs from one of the available templates,
+or by copying an existing job.
+
+You can carefuly prepare your job and the corresponding job script via the \emph{Job Options} button
+and by editing the job script (see lower right).
+
+Don't forget to actually submit your job to the system via the green \emph{Submit} button!
+
+\paragraph{Job templates}
+
+In addition, you can inspect provided job templates, copy them or even create your own templates via the \emph{Templates} button on the top:
+
+\begin{center}
+    \includegraphics*[scale=0.35]{ood_job_templates}
+\end{center}
+
+\subsection{Shell access}
+
+Through the \emph{Shell Access} button that is available under the \emph{Clusters} menu item,
+you can easily open a terminal (shell) session into your VSC account, straight from your browser!
+
+\begin{center}
+    \includegraphics*[scale=0.5]{ood_shell}
+\end{center}
+
+Using this interface requires being familiar with a Linux shell environment (see Appendix~\ref{ch:useful-linux-commands}).
+
+To exit the shell session, type \lstinline|exit| followed by \emph{Enter} and then close the browser tab.
```

Comment:
@wdpypere @baverhey what happens when a user doesn't exit before exiting the shell. i hope this gets cleaned up on the loginnode

Reply:
actually, what happens if you reconnect after not exit? do you get to your existing shell, so sort of screen/tmux thingie?

Reply:
@stdweird there is no screen or tmux or anything. if you leave the session or close the tab in any way, it is gone. no reconnecting is possible.

Reply:
@stdweird there is no screen or tmux or whatever. it really works a bit like an actual ssh client. if you close the session or close the tab in any way your connection is broken and cleaned up. no reconnecting possible to the existing shell, it does not exist. the shell app works quite nice.

Reply:
I clarified this a bit in the text.